### DISCUSSION TITLE
Not working on Qnap with Docker!?

### DISCUSSION THREAD

**Discussion by Kovah** [Kovah/LinkAce#373] [Support]
This topic continues the discussion of a post from the old community forum: "Not working on Qnap with Docker!?" by drallas
A backup of the full conversation can be viewed on the [Internet Archive](https://web.archive.org/web/20220130215430/https://community.linkace.org/t/not-working-on-qnap-with-docker/147).

---

> I followed the simple Docker installation guide to the letter using:
> - an unmodified docker-compose.yml
> - .env file with only changed db / redis pwd values and (chmod 666 set)
> 
> ### linkace_db_1 
> After i start the stack i see the DB container log : :disappointed_relieved:
> 
> ```
> 2022-01-05 17:52:47+00:00 [Note] [Entrypoint]: Entrypoint script for MariaDB Server 1:10.5.13+maria~focal started.
> 2022-01-05 17:52:47+00:00 [Note] [Entrypoint]: Switching to dedicated user 'mysql'
> error: exec: "/usr/local/bin/docker-entrypoint.sh": stat /usr/local/bin/docker-entrypoint.sh: permission denied
> ```
> 
> the linkace_db_1 is also constaly restarting! 
> 
> ### linkace_app_1 
> when i run the cmd to generate the key is see :  :no_mouth: 
> 
> ```
> docker exec linkace_app_1 php artisan key:generate
> **************************************
> *     Application In Production!     *
> **************************************
> 
>  Do you really wish to run this command? (yes/no) [no]:
>  > Command Canceled!
> ```
> It's selecting [no] for me, without an option to click yes! 
> 
> linkace_app_1 container is running fine, but is trowing a http 500 error, i assume because the database is not availible?!
> 
> Little bit stuck now, any help how to solve / approach this would be welcome!

**Comment by Drallas** [comment]
I had qnap support look at my NAS, they reported back that docker and dockers .qpkg permission where altered. I'm sure i kept mine default, so it must have been a process or unsigned .qpkg that altered it. 

Bit busy now, but once i have time i will test again and report back.

**Comment by MZ-Bendlin** [comment]
i have the same problem with linkace on qnap.

I've already contacted Qnap support, unfortunately they haven't been able to help me yet. Statement from Qnap is that it is not an authorization problem.

@Drallas have you already found out what the Qnap Support has adjusted for you?

Thank you for your support.
